bouncer/shared/cf_governance.ts

```diff
@@ -19,9 +17,10 @@ export async function submitGovernanceExtrinsic(
   extrinsic: SubmittableExtrinsic<'promise'>,
   preAuthorise = 0,
 ) {
-  return snowWhiteMutex.runExclusive(async () =>
-    chainflip.tx.governance
+  return snowWhiteMutex.runExclusive(async () => {
+    await using chainflip = await getChainflipApi();
+    await chainflip.tx.governance
```

Comment:
in our previous iteration, this `using` statement was outside this closure in the main function. I believe because it returns immediately that the resource would get cleaned up before the extrinsic was submitted in some cases (e.g. the connection was disconnected while a reference to the closed connection was retained in this closure)